server/src/main/java/org/elasticsearch/index/IndexVersions.java

```diff
@@ -92,8 +92,10 @@ private static IndexVersion def(int id, Version luceneVersion) {
     public static final IndexVersion ES_VERSION_8_12 = def(8_500_004, Version.LUCENE_9_8_0);
     public static final IndexVersion NORMALIZED_VECTOR_COSINE = def(8_500_005, Version.LUCENE_9_8_0);
     public static final IndexVersion UPGRADE_LUCENE_9_9 = def(8_500_006, Version.LUCENE_9_9_0);
-    public static final IndexVersion ES_VERSION_8_13 = def(8_500_007, Version.LUCENE_9_9_0);
+    public static final IndexVersion ES_VERSION_TRANSIENT = def(8_500_007, Version.LUCENE_9_9_0);
     public static final IndexVersion UPGRADE_LUCENE_9_9_1 = def(8_500_008, Version.LUCENE_9_9_1);
+    public static final IndexVersion ES_VERSION_8_13 = def(8_500_009, Version.LUCENE_9_9_1);
```

Comment:
@thecoop I believe this change is what you proposed in a separate PR, right? (or have I misinterpreted)

Reply:
Yes, this is correct